core/server/influx-metrics.js

```diff
@@ -0,0 +1,72 @@
+'use strict'
+
+const request = require('request')
+const { promClientJsonToInfluxV2 } = require('./metrics/format-converters')
+
+module.exports = class InfluxMetrics {
+  constructor(metricInstance, instanceMetadata, config) {
+    this._metricInstance = metricInstance
+    this._instanceMetadata = instanceMetadata
+    this._config = config
+  }
+
+  async registerMetricsEndpoint(server) {
+    server.route(/^\/metrics-influx$/, (data, match, end, ask) => {
+      ask.res.setHeader('Content-Type', 'text/plain')
+      ask.res.end(this.metrics())
+    })
+  }
+
+  async startPushingMetrics() {
+    const sendMetrics = () => {
+      const auth = {
+        user: this._config.username,
+        pass: this._config.password,
+      }
+      request.post(
+        {
+          uri: this._config.uri,
+          headers: { 'Content-Type': 'application/x-www-form-urlencoded' },
+          body: this.metrics(),
+          timeout: this._config.timeoutMillseconds,
+          auth,
+        },
+        function(error, response, body) {
+          if (error) {
+            console.log(
+              `Cannot push metrics. Cause: ${error.name}: ${error.message}`
+            )
+          }
+          if (response && response.statusCode >= 300) {
+            console.log(
+              `Cannot push metrics. ${response.request.href} responded with status code ${response.statusCode}`
+            )
+          }
```

Comment:
If we're just going to log this, we should probably use `console.error()`

That said, we don't really have meaningful access to logs at the moment and this seems like something where if it happens once or twice, we don't really care but if it happens more than N times in a row, we want to know about it. How about keeping an error counter which resets to zero on a successful push and we count more than N errors, throw an exception so we log it to sentry?

Reply:
I agree, `console.error` is better. How about using https://github.com/badges/shields/blob/4ee26bfcdb8336812c85fb4ac72ebd2c1f76343d/core/server/log.js#L32-L37
which uses `console.error()` and sends exception to Sentry? I would like to start with something simpler by sending all errors to Sentry. 
Are you worried that we will sent too many exceptions to Sentry?
As a next step I would like to add metrics for the number of failed requests and for the response time (for the code above).

Reply:
I'm not too worried about the volume of notifications I just think its inevitable that occasionally we'll hit the timeout and generate a false-positive, but I think you're right - lets keep it simple for now. We can manage the notifications in sentry using "ignore until this occurs another N times" or whatever.
Lets use the `error()` helper as you suggest :+1:

Reply:
done